### PR TITLE
LateNight: Fix keylock icon colors

### PR DESCRIPTION
**Pull request by Holzhaus**
Selecting the rectangle in Inkscape, then removing path effects + simplify + reverse did the trick.

### CODE REVIEW

File: res/skins/LateNight/palemoon/buttons/btn__keylock.svg [hunk 1]
```diff
@@ -25,11 +25,9 @@
   <defs
      id="defs6" />
   <path
+     id="rect4195"
      style="fill:#3d3d3c;fill-opacity:1;stroke:none;stroke-opacity:1"
-     fill="#766d65"
```
Comment (Holzhaus) [inline res/skins/LateNight/palemoon/buttons/btn__keylock.svg:29]: I guess this was the culprit :D

**Comment by ronso0** [comment]
straaange,.. I didn't ever use path effects.
I'm using inkscape 1.0 (snap) and maybe it didn't clear the artifacts from those old files. IIRC those were also 'optimzed SVG' at some stage.

Still don't get why it looks okay here. Qt 5.12.8 btw

**Comment by ronso0** [comment]
Looks exactly as before on my machine.

**Comment by Holzhaus** [comment]
> straaange,.. I didn't ever use path effects.
> I'm using inkscape 1.0 (snap) and maybe it didn't clear the artifacts from those old files. IIRC those were also 'optimzed SVG' at some stage.
> 
> Still don't get why it looks okay here. Qt 5.12.8 btw

I don't know if it was a path effect, I think simplify path probably did the trick. I did all tree changes in one go and it worked. Before, I tried to only change the fill color, but that didn't work.

If you look at the diff, there were two contradicting fill colors set on the path (style and fill attributes). Apparently Qt 5.12 prefers the fill values from the style attribute, but Qt 5.15 prefers the one from the fill attribute.

**Comment by ronso0** [comment]
> If you look at the diff, there were two contradicting fill colors set on the path (style and fill attributes). Apparently Qt 5.12 prefers the fill values from the style attribute, but Qt 5.15 prefers the one from the fill attribute.

I saw that, and I guess it's residue from switching between optimized SVG (external plugin) and Plain SVG (built-in) and Inkscape didn't remove the invisble CSS style attributes:
image

**Pull request merged**